src/Utilities/functions.jl

```diff
@@ -4,20 +4,20 @@ using Compat.Test
 """
     evalvariables(varval::Function, f::AbstractFunction)
 
-Returns the value of function `f` if each variable index `vi` is evaluate as `varval(vi)`.
+Returns the value of function `f` if each variable index `vi` is evaluated as `varval(vi)`.
 """
 function evalvariables end
 evalvariables(varval::Function, f::MOI.SingleVariable) = varval(f.variable)
 evalvariables(varval::Function, f::MOI.VectorOfVariables) = varval.(f.variables)
-evalvariables(varval::Function, f::MOI.ScalarAffineFunction) = sum(evalterm.(varval, f.terms)) + f.constant
+evalvariables(varval::Function, f::MOI.ScalarAffineFunction) = mapreduce(t->evalterm(varval, t), +, zero(f.constant), f.terms) + f.constant
 function evalvariables(varval::Function, f::MOI.VectorAffineFunction)
     out = copy(f.constants)
     for t in f.terms
         out[t.output_index] += evalterm(varval, t.scalar_term)
     end
     out
 end
-evalvariables(varval::Function, f::MOI.ScalarQuadraticFunction) = sum(evalterm.(varval, f.affine_terms)) + sum(evalterm.(varval, f.quadratic_terms)) + f.constant
+evalvariables(varval::Function, f::MOI.ScalarQuadraticFunction) = mapreduce(t->evalterm(varval, t), +, zero(f.constant), f.affine_terms) + mapreduce(t->evalterm(varval, t), +, zero(f.constant), f.quadratic_terms) + f.constant
```

Comment:
Same comment about long one-line functions. Prefer to split it up and fit within 80 characters.